vendor/github.com/Azure/azure-quickstart-templates/201-vm-winrm-windows/azuredeploy.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
I made changes to the files, as there are a couple of bugs. 

- makecert is generating negative serial numbers ocassionally
- WinRM was not setup for basic auth.

Both of the above are required for the golang WinRM client to connect. Submitted upstream, not merged yet. When it is merged, I will remove the vendored version.